doc/doxygen/tutorial/tutorial.h.in

```diff
@@ -267,9 +267,9 @@
  *       <td>step-21</td>
  *       <td> The time dependent two-phase flow in
  *       porous media. Extensions of mixed Laplace discretizations. More
- *       complicated block solvers. Simple time stepping.
+ *       complicated block solvers. Simple time stepping using DiscreteTime.
```

Comment:
I think that we have, for the most part, managed to keep the descriptions of the tutorials separated from the classes that have been used in them; we've kept the keywords section for that. I'd be keen to keep this line as it was.

Reply:
Yes, let's drop this here. But it's worth mentioning in the `DiscreteTime` class that step-21 uses it!

Reply:
I fixed it here and I added a mention to step-21 in `discrete_time.h`